hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/pipeline/PipelineReportHandler.java

```diff
@@ -120,4 +116,24 @@ private void processPipelineReport(PipelineReport report, DatanodeDetails dn,
       }
     }
   }
+
+
+  protected void reportDatanode(Pipeline pipeline, DatanodeDetails dn)
```

Comment:
For cleaner semantics, this can be renamed to setReportedDatanode.

Reply:
Done